### PR TITLE
Feature/startup screen

### PR DESCRIPTION
**Pull request by mephi13**
stuff

### CODE REVIEW

File: client/src/client/echo_client.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def run(self) -> None:
                     certpath=self.certpath,
                     friends=[
                         # loneliness
-                    ],
+                    ],  # type: ignore
```
Comment (aCinal) [inline client/src/client/echo_client.py:50]: "friends" is now a set, please change to "set()"

File: client/src/client/user_interface/__init__.py [hunk 2]
```diff
@@ -200,6 +284,84 @@ def on_system_message_received(
...
+            f"or type /friends to see your friendslist.",
+            title="Welcome!",
+            input_validation_function=None,
+        )
+
+        self.view.layout.add(welcome_screen)
+        await self.view.render_all()
+
+    def first_startup_prompt(self, state: StartupState) -> str:
+        """
```
Comment (aCinal) [inline client/src/client/user_interface/__init__.py:341]: Do linters not have a problem with a docstring not in line with the opening quotes?

File: client/src/client/user_interface/tiles.py [hunk 3]
```diff
@@ -211,6 +239,74 @@ async def render(self, t: Terminal) -> None:
...
+        # Validation function should return a boolean
+        # Validation requirements should be stated in the prompt_text
+        self.input_validation = input_validation_function
+
+    async def render(self, t: Terminal) -> None:
+        """Render the Prompt Tile."""
+        await self.clear_tile(t)
+
+        # golden ratio :O
+        phi = 1.618033988
```
Comment (aCinal) [inline client/src/client/user_interface/tiles.py:272]: :0

File: client/src/client/user_interface/__init__.py (hunk 2 above)
Comment (mephi13) [reply client/src/client/user_interface/__init__.py:341]: Nope, they seem to like it if it's multiline

**Review by aCinal — APPROVED** [review]

**Pull request merged**